packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -345,6 +346,8 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
 	 */
 	protected abstract onDisconnect();
 
+	protected abstract get serializer(): IFluidSerializer;
+
 	/**
 	 * Submits a message by the local client to the runtime.
 	 * @param content - Content of the message
```

Comment:
Mention something like this?

```suggestion
	 * @param content - Content of the message
	 * Note: handles contained in the message object should not be encoded in any way
```